code/game/gamemodes/demon/demoninfo.dm

```diff
@@ -0,0 +1,397 @@
+#define POWERUPTHRESHOLD 3 //How many souls are needed per stage.
+
+#define BASIC_DEVIL 0
+#define BLOOD_LIZARD 1
+#define TRUE_DEVIL 2
+#define ARCH_DEVIL 3
+
+#define SOULVALUE soulsOwned.len-reviveNumber
+
+#define DEVILRESURRECTTIME 600
+
+var/global/list/allDevils = list()
+var/static/list/lawlorify = list (
```

Comment:
Okay, I'll change it, thought I still don't understand the difference between when to use static and global.